### DISCUSSION TITLE
Overlay not showing when beacon is disabled

### DISCUSSION THREAD

**Discussion by pospetur** [gilbarbara/react-joyride#661] [Q&A]
Hello hello,
I have disabled the beacon in my first step as shown below
`const steps = [
            {
                disableBeacon: true,
...
            }
        ]
`
but when I do it the overlay will not show. If I change it to "false" the overlay will show. Is this a bug or am I missing something in the docs?

**Comment by gilbarbara** [comment]
Please post a codesandbox with a minimal reproducible example
